packages/@okta/vuepress-site/docs/guides/configure-claims-sharing/main/oktaoidc/policies.md

```diff
@@ -0,0 +1,41 @@
+You can configure many scenarios for authentication using claims sharing and policies in your Okta SP and Okta IdP orgs.
+
+### Authentication policy example
+
+[Create an authentication policy and rule for your app](https://help.okta.com/okta_help.htm?type=oie&id=ext-create-auth-policy) in your SP org. Select any two factors, don't select a possession constraint, and allow any authenticators.
+
+With trust claims enabled and your IdP org able to verify any two factors, you can satisfy the requirements in the SP org.
+
+#### Other authentication policy scenarios
+
+* **Possession factor constraints:** If you enable any possession factor constraints in the authentication policy of your SP org, the IdP org must satisfy the requirement with appropriate factor verification.
+
+* **Authentication methods:**
+  * **Allow any method that can be used to meet the requirement**: If you enable this setting in your SP org, you can satisfy the policy requirements by using any authenticator that meets those requirements. This includes authenticators that aren't configured locally in the SP org.
+  * **Disallow specific authentication methods**: If you specify authentication methods to disallow, then the SP org disallows those methods.
+  * **Allow specific authentication methods**: If you specify authentication methods to allow, then the SP org only considers those methods.
+
+  After you define these conditions, if you still haven't met the policy requirement, then the SP org redirects you to verify any locally configured authenticator. If there's no local authenticator available, or the enrollment policy for a particular authenticator is disabled, then the SP org displays an error.
+
+### Global session policy example
+
+This same concept applies to the global session policy. Without trust claims enabled, if you have only the password authenticator configured in the SP org, you can't save a global session policy rule that requires MFA.
+
+However, with trust claims enabled, you can specify MFA as required. As long as the claim is coming from the IdP, the session is established because that claim can satisfy the global session policy rule.
```

Comment:
```suggestion
However, with trust claims enabled, you can specify MFA as required. As long as the claim comes from the IdP, the session is established because that claim can satisfy the global session policy rule.
```